src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs

```diff
@@ -588,8 +589,17 @@ public FormulaType GetNodeType(TexlNode node)
 
         // Called by language server to get custom tokens.
         // If binding is available, returns context sensitive tokens.  $$$
+        // Keeping this temporarily for custom publish tokens notification

```

Comment:
I do have another user story to revisit Publish Tokens and clean this up later on